src/@commonsku/styles/Toggle.tsx

```diff
@@ -100,19 +100,25 @@ const ToggleLink = styled.a<{selected?: boolean, stretch?:boolean, size?: Toggle
     ${SharedStyles}
   }`
 
-type ToggleProps = React.PropsWithChildren<{
+type ToggleProps = {
   stretch?: boolean;
   onClick?: React.MouseEventHandler<HTMLDivElement>;
   size?: ToggleSize;
   style?: React.CSSProperties;
-} & SharedStyleTypes>;
+  children?: (string | number | boolean | React.ReactElement<any, string | React.JSXElementConstructor<any>>)[];
```

Comment:
I think you can use `children: React.ReactNode` for the `children` prop.   I think it's defined as
```
type ReactNode =
  | ReactElement
  | string
  | number
  | Iterable<ReactNode>
  | ReactPortal
  | boolean
  | null
  | undefined;